File: Aman_Bhaiya_DSA_List/Arrays/LQ33.py
# print largest number by rearranging elements of list

def largest_number(nums):

    if len(nums) == 1:
        return str(nums[0])

    for i in range(len(nums)):
        for j in range(i+1, len(nums)):
            if int(str(nums[i])+str(nums[j])) < int(str(nums[j])+str(nums[i])):
                nums[i], nums[j] = nums[j], nums[i]
    return ''.join(map(str, nums))

nums = [3, 30, 34, 5, 9]
print(largest_number(nums))

File: Aman_Bhaiya_DSA_List/Arrays/LQ33.java
package Aman_Bhaiya_DSA_List.Arrays;

import java.util.Collection;
import java.util.Collections;
import java.util.List;

class LQ33{
    //print largest number by arranging elements of array

    public static void printLargest(List<String> arr){
        Collections.sort(arr, (X, Y) -> (Y + X).compareTo(X + Y));
        for(String s:arr){
            System.out.print(s);
        }
    }

    public static void main(String[] args) {
        List<String> arr = List.of("54", "546", "548", "60");
        printLargest(arr);
    }
}